apollo-router/src/configuration/mod.rs

```diff
@@ -273,9 +273,8 @@ impl fmt::Display for ListenAddr {
 pub struct Cors {
     #[serde(default)]
     #[builder(default)]
-    /// Set to false to disallow any origin and rely exclusively on `origins`.
+    /// Set to true to allow any origin.
     ///
-    /// /!\ Defaults to true
```

Comment:
Why not retain this line to note that:
```
/// /!\ Defaults to false
```
?

Reply:
Fair! I'll remove the /!\ however, since it s not a security concern anymore 😅